### ISSUE TITLE
[Accessibility]: a button is incorreclty contained in a div with role=button 

### ISSUE THREAD

**Issue by philljenkins** [carbon-design-system/carbon-charts#1560]
### Name

Phill Jenkins

### Are you an IBM employee?

- [X] Yes

### Environment

Automated testing tool and ruleset

### What happened? What did you expect to happen instead?

Tab key should only stop once on a button,
Multiple buttons are defined on a chart control.
The Accessibility Checker flags the issue

### What WCAG 2.1 checkpoint does the issue violate?

2.1.1 Keyboard

### Version

latest

### Data & options used

```js
4-4-2023
```

### Relevant log output

```js
The behavior can be observed here: https://ibm.box.com/s/lfa8ci0pnbm617j8jrkaif28btm384yx
```

### Codesandbox example

Review the screen recording here: https://ibm.box.com/s/lfa8ci0pnbm617j8jrkaif28btm384yx

### What priority level would this issue be in your opinion?

P2

**Comment by philljenkins** [comment]
we see a div with role=button, which contains a button element, and a there should only be one:
![image](https://github.com/carbon-design-system/carbon-charts/assets/16193609/a61173a3-6fc2-4305-a01b-bf013c83ed31)
